gravitee-apim-gateway/gravitee-apim-gateway-services/gravitee-apim-gateway-services-healthcheck/src/test/java/io/gravitee/gateway/services/healthcheck/http/AbstractManagedEndpointRuleHandlerTest.java

```diff
@@ -57,7 +59,7 @@ public abstract class AbstractManagedEndpointRuleHandlerTest {
     private TemplateEngine templateEngine;
 
     @RegisterExtension
-    static WireMockExtension wm = WireMockExtension.newInstance().options(wireMockConfig().dynamicPort()).build();
+    static WireMockExtension wm = WireMockExtension.newInstance().options(wireMockConfig().dynamicPort()).proxyMode(true).build();
```

Comment:
Required to mock other domain than localhost
See https://wiremock.org/docs/junit-jupiter/#proxy-mode